docs/manual/shared_memory.rst

```diff
@@ -55,6 +55,7 @@ Save the following xml as cyclonedds.xml.
               <SubQueueCapacity>256</SubQueueCapacity>
```

Comment:
I think we should add in this documentation which settings have default values and what these are (if any).

For instance, the setting SubHistoryRequest will fall back to the value 16 if nothing is set in the configuration file.

Reply:
True, I set the current maximum value for all of them explicitly. Especially for queue size this makes sense I think to reduce the chance of data loss. Note that the majority of shared memory is only used if the queue actually contains data, since the queue itself just stores pointer equivalents.

We are currently limited by some hardcoded values of iceoryx. This will likely be improved. Documentation will be a separate PR.

Reply:
Will be done in a documentation PR. Probably best once the features are complete.